auth/src/main/res/layout-land/auth_method_picker_layout.xml

```diff
@@ -4,7 +4,7 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:clipToPadding="false"
-    android:background="@color/white"
+    android:background="@color/gray_1"
```

Comment:
Don't use any background - in this case windowBackground is used, which is already gray. Be careful about this, setting multiple overlapping backgrounds on a screen is hard on performance - AS gives warnings about it.